.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__/actuation_computer.cpython-38.pyc
```

Comment:
I think you can just add the entire folder here: `__pycache__` to avoid having to add all files.

Reply:
Done